.github/ISSUE_TEMPLATE/release-check-list.md

```diff
@@ -30,45 +30,43 @@ assignees: ''
+          before making sure this step works as expected.
+        * attestation-agent and td-shim
+          * https://github.com/kata-containers/kata-containers/blob/CCv0/versions.yaml
+            * Change the version
 
-- [ ] - 6. Update Kata Containers to use the latest td-shim
+- [ ] - 5. Wait for kata-runtime-payload-ci to be successfully built
+        * After the previous PR is merged wait for the kata-runtime-payload-ci (https://github.com/kata-containers/kata-containers/actions/workflows/cc-payload-after-push.yaml) has completed, so the latest kata-runtime-payload-ci containers the changes
 
-        * https://github.com/kata-containers/kata-containers/blob/CCv0/versions.yaml
-          * Change the version
-
-- [ ] - 7. Check if there are new changes in the pre install payload script
+- [ ] - 6. Check if there are new changes in the pre install payload script
 
         * https://github.com/confidential-containers/operator/tree/main/install/pre-install-payload
           * The last commit there must match what's in the following files as preInstall / postUninstall image
             * Enclave CC: https://github.com/confidential-containers/operator/blob/main/config/samples/enclave-cc/base/ccruntime-enclave-cc.yaml
             * Kata Containers:
               Note that for Kata Containers, we're looking for the newTag, below the quay.io/confidential-containers/container-engine-for-cc-payload image
-              * s390x: https://github.com/confidential-containers/operator/blob/main/config/samples/ccruntime/s390x/kustomization.yaml
-              * x86_64: https://github.com/confidential-containers/operator/blob/main/config/samples/ccruntime/default/kustomization.yaml
+              * default: https://github.com/confidential-containers/operator/blob/main/config/samples/ccruntime/default/kustomization.yaml
 
-- [ ] - 8. Ensure the Operator is using the latest CI builds and that the Operator tests are passsing
+- [ ] - 7. Ensure the Operator is using the latest CI builds and that the Operator tests are passsing
 
         * Enclave CC:
           * SIM: https://github.com/confidential-containers/operator/blob/main/config/samples/enclave-cc/sim/kustomization.yaml
           * HW: https://github.com/confidential-containers/operator/blob/main/config/samples/enclave-cc/base/ccruntime-enclave-cc.yaml
           * Note that we need the quay.io/confidential-containers/runtime-payload-ci registry and enclave-cc-{SIM,HW}-latest tags
         * Kata Containers:
-          * s390x: https://github.com/confidential-containers/operator/blob/main/config/samples/ccruntime/s390x/kustomization.yaml
-          * x86_64: https://github.com/confidential-containers/operator/blob/main/config/samples/ccruntime/default/kustomization.yaml
+          * default: https://github.com/confidential-containers/operator/blob/main/config/samples/ccruntime/default/kustomization.yaml
```

Comment:
Just for me to understand the recent changes: the runtime-payload is now multi-arch. The s390x overlay is only needed currently to restrict the runtimeClasses (i.e. makes no sense to have, say, kata-qemu-sev on s390x).

Reply:
Yeah - that was a change I introduced a few weeks ago in https://github.com/confidential-containers/operator/pull/169/commits/44ff27b35bf3917145057afc85810e97b7e72d97